source/tutorials/file-sets.md

```diff
@@ -23,23 +24,27 @@ Using these features directly can be tricky however:
   can address these problems.
   However, it's often hard to express the desired path selection using the `filter` function interface.
 
-In this tutorial you'll learn how to use the Nixpkgs [`fileset` library](https://nixos.org/manual/nixpkgs/unstable/#sec-fileset) to work with local files in derivations.
-It abstracts over built-in functionality and offers a safer and more convenient interface.
+In this tutorial you'll learn how to use Nixpkgs' [`lib.fileset` library](https://nixos.org/manual/nixpkgs/unstable/#sec-fileset) to work with local files in derivations.
+It abstracts over built-in functionality with a safer and more convenient interface.
 
 ## File sets
 
-The file set library is based on the concept of _file sets_,
+The basic concept is that of a _file set_,
```

Comment:
Revert too (or if you want file set as singular, say "based on the concept of a file set". "The basic concept is" - of what? What's the subject? And starting with "the basic concept" even if you say "of the file set library" is probably not even important, and the original sentence said what we want: that the file set revolves around this basic concept.

Reply:
In the docs team meeting right now, changing it to

> A _file set_ is a data type representing a collection of local files.
File sets can be created, composed, and manipulated with the various functions of the library.